sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -525,11 +532,41 @@ def index_documents(self, batch, **kwargs):
         :type batch: IndexDocumentsBatch
         :rtype:  List[IndexingResult]
         """
-        index_documents = IndexBatch(actions=batch.actions)
+        return self._index_documents_actions(actions=batch.actions, **kwargs)
+
+    @distributed_trace
```

Comment:
We generally don't provide spans for internal functions. In this case, it also leads to a nested span even in the non-split case.